docs/presentations/articles/hugging_face.md

```diff
@@ -0,0 +1,54 @@
+# HuggingFace
```

Comment:
It seems they spell themselves `Hugging Face` (with a space). Could you adjust that throughout the document?